src/VAC/View.cpp

```diff
@@ -1359,6 +1148,108 @@ void View::drawBackground_(Background * background, int frame)
              xSceneMin(), xSceneMax(), ySceneMin(), ySceneMax());
 }
 
+void View::drawLine(double x, double y)
+{
+    drawShape(x, y, false, 2);
+}
+
+void View::drawCircle(double x, double y)
+{
+    drawShape(x, y, true);
+}
+
+void View::drawPolygon(double x, double y, int countAngles, double rotation, bool isRemoveVertices)
+{
+    drawShape(x, y, false, countAngles, rotation, isRemoveVertices);
+}
+
+void View::drawShape(double x, double y, bool isCircle, int countAngles, double rotation, bool isRemoveVertices)
```

Comment:
maybe use the shape type instead of bool "isCircle" to make clearer what the variable is for?
(it is hard to see at the call site what bools are, so I prefer enums, and we have an enum for the shape already)

Also, just "removeVertices" is probably better grammar, but having that as an enum with two entries is even clearer, i.e: `enum DrawShapeMode { KeepVertices, RemoveVertices };`

Reply:
Done